nix/pkgs/control-plane/cargo-project.nix

```diff
@@ -71,8 +72,8 @@ let
     GIT_VERSION = "${gitVersions.tag_or_long}";
 
     inherit LIBCLANG_PATH PROTOC PROTOC_INCLUDE;
-    nativeBuildInputs = [ clang pkg-config openapi-generator which git llvmPackages.bintools ];
-    buildInputs = [ llvmPackages.libclang protobuf openssl.dev systemdMinimal.dev utillinux.dev rdma-core ];
+    nativeBuildInputs = [ clang pkg-config paperclip which git llvmPackages.bintools ];
+    buildInputs = [ llvmPackages.libclang protobuf openssl.dev systemdMinimal.dev utillinux.dev ];
```

Comment:
`rdma-core` removal is intentional here? I didn't think that's redundant here.

Reply:
oops, good catch, it was bad rebase, fixed!